src/pages/taskHistory/ColumnHeaders.test.tsx

```diff
@@ -0,0 +1,184 @@
+import { context } from "components/HistoryTable";
+import { HistoryTableReducerState } from "components/HistoryTable/historyTableContextReducer";
+import { taskHistoryMaxLength as maxLength } from "constants/history";
+import {
+  fireEvent,
+  renderWithRouterMatch as render,
+  waitFor,
+} from "test_utils";
+import { string } from "utils";
+import ColumnHeaders from "./ColumnHeaders";
+
+const { trimStringFromMiddle } = string;
+const { HistoryTableProvider } = context;
+const longVariantName =
+  "really_really_really_really_really_really_long_variant_name";
+const trimmedVariantName = trimStringFromMiddle(longVariantName, maxLength);
+
+const initialState: HistoryTableReducerState = {
+  loadedCommits: [],
+  processedCommits: [],
+  processedCommitCount: 0,
+  commitCache: new Map(),
+  currentPage: 0,
+  pageCount: 0,
+  columns: [],
+  historyTableFilters: [],
+  commitCount: 10,
+  visibleColumns: [],
+  columnLimit: 7,
+};
+
+interface wrapperProps {
```

Comment:
[nit] Types and interfaces are capitalized
```suggestion
interface WrapperProps {
```